CROSSSECTOR/DIH/DigitalInnovationHub/schema.json

```diff
@@ -0,0 +1,158 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "$schemaVersion": "0.0.1",
+  "$id": "https://smart-data-models.github.io/dataModel.DIH/DigitalInnovationHub/schema.json",
+  "title": "Sample DIH",
+  "description": "Digital Innovation Hubs are one-stop-shops that help companies to become more competitive with regard to their business/production processes.",
+  "type": "object",

```

Comment:
missing the usual $ref.
without them, you have no id for the data model.
Additionally, you have not included the attribute type which is mandatory for the NGSI.